File: Meadow_DotNet_SDK/Project_Templates/templates/MeadowRaspberryPiCSharp/MeadowApp.cs
using Meadow;
using System.Threading.Tasks;

namespace MeadowApplication.Template;

public class MeadowApp : App<RaspberryPi>
{
    public override Task Initialize()
    {
        Resolver.Log.Info("Initialize...");

        return Task.CompletedTask;
    }

    public override Task Run()
    {
        Resolver.Log.Info("Run...");

        Resolver.Log.Info("Hello, Raspberry Pi!");

        return Task.CompletedTask;
    }
}

File: Meadow_DotNet_SDK/Project_Templates/templates/MeadowCoreComputeVBNet/MeadowApp.vb
Imports System.Threading.Tasks
Imports Meadow
Imports Meadow.Devices

Public Class MeadowApp
    Inherits App(Of F7CoreComputeV2)

    Public Overrides Function Initialize() As Task
        Resolver.Log.Info("Initialize...")

        Return Task.CompletedTask
    End Function

    Public Overrides Function Run() As Task
        Resolver.Log.Info("Run...")

        Resolver.Log.Info("Hello, Meadow Core-Compute!")

        Return Task.CompletedTask
    End Function
End Class